CodeU_Assignement_1/src/codeu_assignement_1/SinglyLinkedList.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package codeu_assignement_1;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+/**
+ *
+ * @author Karine
+ * 
+ * Bounds, index and general interface is as in Java LinkedList JavaDoc
+ * https://docs.oracle.com/javase/7/docs/api/java/util/LinkedList.html
+ * 
+ */
+public class SinglyLinkedList <E> {

```

Comment:
Yes, they asked an implementation of a linked list. 
I used the same interface Java does, so any Java programmer can get what's going on there...